zipline/utils/tradingcalendar.py

```diff
@@ -1,4 +1,3 @@
-#
```

Comment:
@ehebert my understanding was that we don't update copyright unless a new file is added?